pyxdsm/XDSM.py

```diff
@@ -456,23 +468,35 @@ def _build_edges(self):
         h_edges = []
         v_edges = []
 
-        edge_string = "({start}) edge [DataLine] ({end})"
+        edge_string = "({start}) edge [{style}] ({end})"
```

Comment:
This is just me being nitpicky but this is a format string, so the variable name could be improved (since strings with {} are often assumed to be f-strings).

Reply:
Do you mean changing `edge_string` to something like `edge_string_template`? or changing the name of the `{style}` placeholder?

Reply:
I would prefer changing the variable name to `edge_format_string`, since it's a format string; template strings are a different thing.